llama-index-integrations/tools/llama-index-tools-google/llama_index/tools/google/calendar/base.py

```diff
@@ -144,7 +144,7 @@ def create_event(
         location: Optional[str] = None,
         start_datetime: Optional[Union[str, datetime.datetime]] = None,
         end_datetime: Optional[Union[str, datetime.datetime]] = None,
-        attendees: Optional[List[str]] = None,
+        attendees: Optional[List[str]] = [],
```

Comment:
due to how pass-by-reference works, we shouldn't put lists as defaults in functions

Instead, lets check if attendees is none, and convert it to an empty list inside the function body

Reply:
make sense, updated the code.